src/Cli/dotnet/Commands/Tool/Run/ToolRunCommandParser.cs

```diff
@@ -39,6 +42,7 @@ private static Command ConstructCommand()
 
         command.Arguments.Add(CommandNameArgument);
         command.Arguments.Add(CommandArgument);
+
```

Comment:
It seems like you were planning to add one-shot execution to tool run then didn't. Can we revert the changes in this file and move the --yes option to the ToolExecuteCommandParser?